### PR TITLE
Add ai apps workshop

### PR DESCRIPTION
**Pull request by sean1588**
## Description

DO NOT MERGE: I still need to get the hubspot form id before we can ship this.

Add AI apps workshop. 

## Checklist:

- [ ] I have reviewed the [style guide](https://github.com/pulumi/pulumi-hugo/blob/master/STYLE-GUIDE.md).
- [ ] If blogging, I have reviewed the [blogging guide](https://github.com/pulumi/pulumi-hugo/blob/master/BLOGGING.md).
- [ ] I have manually confirmed that all new links work.
- [ ] I added aliases (i.e., redirects) for all filename changes.
- [ ] If making css changes, I rebuilt the bundle.

### CODE REVIEW

**Review by shughes26 — APPROVED** [review]

File: themes/default/content/resources/ai-apps-to-production/index.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+    sortable_date: 2023-12-14T09:00:00.000-08:00
+    # Duration of the webinar.
+    duration: "60 minutes"
+    # Datetime of the webinar.
+    datetime: 
+    # Description of the webinar.
+    description: |
+        What does it take to go from an idea in a notebook to an application handling real-world traffic? In this workshop, the Pinecone and Pulumi teams will explore the infrastructure and service architecture you need to scale AI apps in production. You will learn how to create and manage containerized microservices running on Amazon ECS, the networking and message queue infrastructure connecting the microservices, and the data stores powering the AI app such as Pinecone and Amazon RDS Postgres.
+
+        This workshop is designed to help AI developers and engineers build and scale AI infrastructure. We will guide you through the Pulumi platform with diagrams and a series of labs to help accelerate your AI apps.
```
Comment (aaronkao) [inline themes/default/content/resources/ai-apps-to-production/index.md:65]:
```suggestion
        This workshop is designed to help AI developers and engineers gain valuable insights and practical strategies for evolving AI applications into resilient and efficient cloud-native solutions. We will guide you through the Pulumi platform with diagrams and a series of labs to help accelerate your AI apps.
```

File: themes/default/content/resources/ai-apps-to-production/index.md [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+    youtube_url:
+    # Sortable date. The datetime Hugo will use to sort the webinars in date order.
+    sortable_date: 2023-12-14T09:00:00.000-08:00
+    # Duration of the webinar.
+    duration: "60 minutes"
+    # Datetime of the webinar.
+    datetime: 
+    # Description of the webinar.
+    description: |
+        What does it take to go from an idea in a notebook to an application handling real-world traffic? In this workshop, the Pinecone and Pulumi teams will explore the infrastructure and service architecture you need to scale AI apps in production. You will learn how to create and manage containerized microservices running on Amazon ECS, the networking and message queue infrastructure connecting the microservices, and the data stores powering the AI app such as Pinecone and Amazon RDS Postgres.
```
Comment (aaronkao) [inline themes/default/content/resources/ai-apps-to-production/index.md:63]:
```suggestion
        What does it take to go from an idea in a notebook to an application handling real-world traffic? In this workshop, the Pinecone and Pulumi teams will explore the infrastructure and service architecture you need to scale AI apps in production. We will delve into deploying high-volume AI systems through scalable microservices, efficient data processing, and seamless synchronization between user interfaces and databases. We will examine the nuances of containerization for enhanced portability and Infrastructure as Code (IaC) for streamlined cloud deployments. The workshop will also discuss industry best practices in scalability and security for production-grade AI systems in a cloud-native landscape.
```

File: themes/default/content/resources/ai-apps-to-production/index.md [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+    # The webinar presenters
+    presenters:
+        - name: Scott Lowe
+          role: Principal Community Engineer, Pulumi
+        - name: TBD 
+          role: Pinecone Engineer
+    
+    learn:
+        - The basics of infrastructure as code and the Pulumi programming model
+        - How to provision, update, and destroy AWS resources
```
Comment (aaronkao) [inline themes/default/content/resources/ai-apps-to-production/index.md:76]:
```suggestion
        - How to provision, update, and destroy AWS resources (Amazon ECS clusters, networking infrastructure, messaging queues, and Amazon RDS Postgres databases)
```

**Review by aaronkao — CHANGES_REQUESTED** [review]
Updated the abstract with some comments from Pinecone

**Review by isaac-pulumi — APPROVED** [review]

**Pull request merged**